tests/form-filling.js

```diff
@@ -0,0 +1,129 @@
+const helper = require('./helper');
+const genericFormPo = require('./page-objects/forms/generic-form.po');
+const answeredQuestions = [];
+
+const getPaths = (obj, prevKey, r = []) => {
```

Comment:
I've got this handled with Flatten now. There's still a case where we have repeats in the form which equate to arrays in the report. I'm getting that and removing the index so it can be treated as a key like the rest.  I don't feel like the repeats will be used very frequently above 1 but would require a decent amount of more work to figure out on the UI when there's multiple repeats and then pulling the data out of the form to fill the repeats.